### ISSUE TITLE
hungarian translation

### ISSUE THREAD

**Issue by lovasatt** [abnerfs/cs2-rockthevote#23]
Hi,
Here is the hungarian translation. Cheers
Tom1st
[hu.json](https://github.com/abnerfs/cs2-rockthevote/files/14321780/hu.json)

**Comment by abnerfs** [comment]
Thanks man can you translate those two new phrases as well? Planning to add in the next version
```json
{
  "nextmap": "Next map will be {green}{1}",
  "nextmap.decided-by-vote": "Next map will be decided by vote"
}
```

**Comment by lovasatt** [comment]
{
  "nextmap": "A következő pálya {green}{1} {default}lesz",
  "nextmap.decided-by-vote": "A következő pályáról szavazással döntenek"
}

**Comment by abnerfs** [comment]
Thanks for your contribution but another person sent me the Hungarian translation before and it is merged into main from version v1.6.0 https://github.com/abnerfs/cs2-rockthevote/pull/26
